bin/check-redis-keys.rb

```diff
@@ -68,6 +68,12 @@ class RedisKeysCheck < Sensu::Plugin::Check::CLI
          required: false,
          default: '*'
 
+  option :conn_failure_status,
+         long: '--conn-failure-status EXIT_STATUS',
+         description: 'Exit status for Redis connection failures',
+         default: 'unknown',
+         in: %w(unknown warning critical ok)
```

Comment:
I question the value of `ok` as an option...A connection failure is clearly not OK...